vscode/src/client/analyzerClient.ts

```diff
@@ -11,6 +11,7 @@ export class AnalyzerClient {
   private config: vscode.WorkspaceConfiguration | null = null;
   private extContext: vscode.ExtensionContext | null = null;
   private analyzerServer: ChildProcessWithoutNullStreams | null = null;
+  private rpcServer: ChildProcessWithoutNullStreams | null = null;
```

Comment:
I think we should use only the analyzerServer. That is, if we start kai/rpc-server, that process should be assigned to analyzerServer and be used interchangeably.

The changes in how we interact with that server have to do with whether genAI is enabled or not.